keyboards/tr60w/readme.md

```diff
@@ -0,0 +1,26 @@
+# TR60W
+
+[![TR60W Imgur Link](https://imgur.com/a/vezmI99/)](https://imgur.com/a/vezmI99/)
+
+This is the hotswap pcb supplied by Triangle Lab with their tr60 case.
+
+* Keyboard Maintainer: [Joule-Flow](https://github.com/Joule-Flow)
+* Hardware Supported:
+  * ATMEGA 32u4 supported
+  * The pcb does seem to be designed by fox labs
+  * 4 rgb leds around the escape key socket
+  * kalih hot swap sockets
+  * north facing switches
+  * usb c receptacle
+  * smd nonrgb-in-switch leds
```

Comment:
Lines 8-15 should be replaced with just:

```Markdown
* Hardware Supported: TR60W PCB, ATmega32U4
```

Reply:
done.